test/integration/targets/nxos_bgp/tests/common/sanity.yaml

```diff
@@ -4,10 +4,13 @@
   when: ansible_connection == "local"
 
 - set_fact: neighbor_down_fib_accelerate="true"
-  when: not titanium
+  when: (not titanium) and (imagetag and ((imagetag | version_compare('N1', 'ne')) and (imagetag | version_compare('D1', 'ne'))))
 
 - set_fact: reconnect_interval="55"
-  when: not titanium
+  when: (not titanium) and (imagetag and ((imagetag | version_compare('N1', 'ne')) and (imagetag | version_compare('D1', 'ne'))))
+
+- set_fact: isolate="false"
+  when: not (platform | match("N35"))
```

Comment:
Please use `is` instead of filter. For e.g https://github.com/ansible/ansible/blob/devel/test/integration/targets/prepare_nxos_tests/tasks/main.yml#L62